inabox/tests/integration_suite_test.go

```diff
@@ -117,13 +118,21 @@ var _ = BeforeSuite(func() {
 	testConfig = deploy.NewTestConfig(testName, rootPath)
 
 	if testConfig.Environment.IsLocal() {
+		// Create a shared Docker network for all containers
+		nw, err := network.New(context.Background(),
+			network.WithDriver("bridge"),
+			network.WithAttachable())
+		Expect(err).To(BeNil(), "failed to create Docker network")
+		logger.Info("Created Docker network", "name", nw.Name)
```

Comment:
is this just going to create a random named docker network? Looks like you never delete it anywhere iiuc? So when test fails or inabox mode is launched, how do we delete these networks?

Reply:
Yeah seems random
```
Sep 19 11:54:09.125 INF tests/integration_suite_test.go:128 Created Docker network name=194e306d-8eed-4366-b077-3f602811607a
```

c87c7fbb413217e831d589ebe65ae92358cd32f5